### ISSUE TITLE
[Sovereign] 6p +2 invisible mastery points after already being maxed out 

### ISSUE THREAD

**Issue by Sciberr** [WarEmu/WarBugs#17040]
An introduction:

The +2 mastery point bonus gives extra invisible points even when already maxed out in a mastery path

Details:

I have no idea if this is intentional so those points don't go to waste...

Evidence:

https://youtu.be/v8jOlF1ZGwQ

Additional Information:

N/A

How to reproduce the bug:

Max out a mastery path normally then equip 6p sov to get an extra 2 invisible points.

**Comment by wargrimnir** [comment]
Does this result in the same level of ability strength? Losing two points when you don't have the M4 should result in a loss of ability strength if it scales from points in the tree. This may just be the training window not updating correctly. It makes a significant difference where to track the issue from.

**Comment by Sciberr** [comment]
yes i used my default mastery points to max out grace path first to get the maximum power out of the paths abilities. THEN if you equip 6p sov for the +2 mastery point bonus it will ADD more power to the paths abilities even though its already maxed. For example max grace is an 80 wounds buff but with the sov gear +2 mastery point bonus it boosts it up to 84 wounds.
